src/enum_variant_accessor.rs

```diff
@@ -489,7 +487,7 @@ pub fn impl_enum_accessor(input: DeriveInput) -> TokenStream {
                 .into_compile_error();
         }
 
-        let span = accessor.span;
+        let span = accessor.alias.span();
```

Comment:
name collisions will highlight the alias is one is provided - or the simple name if there is none